docs/current/cookbook/snippets/cache-dependencies/index.mjs

```diff
@@ -0,0 +1,28 @@
+import { connect } from "@dagger.io/dagger"
+
+connect(
+  async (client) => {
+    // use a node:18 container
+    // mount the source code directory on the host
+    // at /src in the container
+    // mount the cache volume to persist dependencies
+    const source = client
+      .container()
+      .from("node:18")
+      .withDirectory("/src", client.host().directory("."))
+      .withMountedCache(
```

Comment:
In my opinion, the `withWorkdir` call should be in the source.
This might also help on  `withMountedCache`, I'm not sure path are relative but that would be nicer if you could just write `./node_modules`

Feel free to update it on Go and Python if that changes is relevant

Reply:
Done